### PR TITLE
Update paper.md

### PR DESCRIPTION
**Pull request by ljgarcia**
Minor changes  related to British vs American spelling, acronyms and typos

### CODE REVIEW

File: projects/36/bhxiv/paper.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ pasting above link (or yours) in
...
...ring phenotypic information that facilitates communication within the research and clinical genomics communities [@jacobsen2022a]. The Observational Medical Outcomes Partnership (OMOP) Common Data Model allows for large-scale analysis of distributed data to generate evidence for research that promotes better health decisions and better care [@overhage2012a]. These gathered data is used by epidemiologists to monitor the infection, model it and make outbreak analysis and predictions to evaluate policy interventions. To harness Machine Learning (ML) and Artificial Intelligence (AI) approaches to discover meaningful patterns in epidemic outbreaks, we need to ensure that data are FAIR, i.e. data and metadata are accessible and actionable by machines. To leverage data for federated learning/analytics, datasets can be discovered in FAIR Data Points; FAIR data repositories that publish human- and machine-readable metadata for data resources. This project aims to enhance interoperability between health and research data by mapping Phenopackets and OMOP and representing COVID-19 metadata using the FAIR principles to enable discovery, integration and analysis of genotypic and phenotypic data.
```
Comment (NuriaQueralt) [inline projects/36/bhxiv/paper.md:107]: Hi @ljgarcia thank you so much for your review and PR. Is it correct that in this paragraph you only did one change which is: 'The OMOP model' to 'The Observational Medical Outcomes Partnership (OMOP) Common Data Model'? If yes, this is correct and a good call! is it possible for you to add the acronym for common data model such as 'The Observational Medical Outcomes Partnership (OMOP) Common Data Model (CDM)'? In that way we are consistent. Then I will accept your suggestion. Thank you!

File: projects/36/bhxiv/paper.md [hunk 2]
```diff
@@ -253,36 +253,36 @@ Table 1 shows an overview of the Phenopacket entity to OMOP table mappings. The
...
+At the BioHackathon Europe 2021 we discussed some expectations and needs around FAIR ML for biomedical research and clinical data. Regarding expectations, we envision an improvement in aspects such as efficiency, transparency and trust, all thanks to the addition of rich metadata to ML approaches. Efficiency can be improved with metadata supporting, e.g., (semi)automatics runs of the workflows behind ML models or testing over sample data. Transparency can be improved with metadata clearly describing, e.g., (hyper)parameters, split of the training, validation and test sets, measures to ensure independence, and precision and accuracy obtained. Trust would be improved thanks to the transparency. For the case of supervising ML in biology, there are already some recommendations in this regard, taking into account data, optimisation, model and evaluation (DOME recommendations) [@walsh2021a], although not yet translated into structured metadata. The translation to structure metadata should hopefully make it easier to apply them in a coherent way so models can be compared to each other under the same parameters.
```
Comment (NuriaQueralt) [inline projects/36/bhxiv/paper.md:280]: Hi @ljgarcia I cannot see what is your change(s) suggestion here. Could you clarify? Thanks

File: projects/36/bhxiv/paper.md [hunk 3]
```diff
@@ -293,29 +293,29 @@ To overcome this situation, two methods can be developed:
...
...L format for its input data and applies ML methods for embedding ontologies. It generates output datasets that can further be used as features for other ML tasks such as classification and clustering. MOWL has already identified some generic metadata for ML and has aligned it to [Schema.org](https://schema.org/). Schema.org is a community-based project on schemas for structured data on the Internet, e.g., on web pages. MOWL metadata specification aims to cover those elements from ML datasets common to multiple domains, lowering the barrier for researchers to adopt it. The specification includes overview information (identification, name, description, keywords) provenance (creator, contributor, publisher), links to others (citation) and elements aiming to enable reusability (measurement technique, schema or standard it is based on, where it has been evaluated, distribution, size). Another example in this line, that could be adapted for the ML case, is the [computational workflow specification](https://bioschemas.org/types/ComputationalWorkflow/1.0-RELEASE) provided by [Bioschemas](https://bioschemas.org/), a community extending and tailoring Schema.org for the bioinformatics domain.
```
Comment (NuriaQueralt) [inline projects/36/bhxiv/paper.md:297]: Hi @ljgarcia I cannot see what is your change(s) suggestion here. Could you clarify? Thanks

File: projects/36/bhxiv/paper.md [hunk 4]
```diff
@@ -293,29 +293,29 @@ To overcome this situation, two methods can be developed:
...
...ed by the community, generated mapping queries in SQL, and created a prototype for a Phenopacket builder. Second, we stemmed work to map both standards to the Semantic Web, in particular we started modelling semantic Phenopackets incorporating SPHN requirements and working on the ETL transformation of OMOP to the semantic model of the rare disease patient registry data in EJP RD, which will enable mapping to semantic Phenopackets. Third, we started discussing FAIR federated ML and we detected common expectations and needs. We did this with the enriching experience of exchanging different strategies utilised among different academia and industry organisations. Last, we recognised the FAIRification of ML is indeed a challenging issue that requires a broader community discussion, involving several stakeholder representatives to facilitate agreements on standards such as the DOME recommendations or Schema.org, technologies such as OWL, infrastructure such as FAIR Data Points, and evaluation methods such as benchmarks, and arguably, more events like the Biohackathon to support all of these endeavors. It is worth highlighting that these outcomes are the seed for different collaborations.
```
Comment (NuriaQueralt) [inline projects/36/bhxiv/paper.md:301]: Hi @ljgarcia I cannot see what is your change(s) suggestion here. Could you clarify? Thanks

File: projects/36/bhxiv/paper.md [hunk 5]
```diff
@@ -293,29 +293,29 @@ To overcome this situation, two methods can be developed:
...
...ions about the presented projects. Such discussions provided invaluable inputs for the ongoing and future work of the projects, while offering participants an effective way to broaden their knowledge of the emerging research trends and technologies and to start collaborations with other researchers in academia and industry. This also came with some challenges as the hybrid meeting required communication across the different intercontinental time zones. Our project had 17 participants (10 onsite and 7 online) from different countries and projects/organisations such as the UK, US, NL, SP or CH. We engaged with 10 participants that were new to the project, and 10 that were new to the BioHackathon-EU. We interacted with the #17 BHEU project to discuss GA4GH Beacon API interoperability with GA4GH Phenopackets data standard and the #27 and #30 BHEU projects and with the OWKIN company to discuss FAIR for ML, with the N3C consortium in the US and EGA in EU to provision COVID-19 datasets, with the SPHN project another health data project based on Semantic Web Technologies and FAIR, and we investigated the interoperability of data represented as semantic Phenopackets with the EJP RD project.
```
Comment (NuriaQueralt) [inline projects/36/bhxiv/paper.md:303]: Hi @ljgarcia I cannot see what is your change(s) suggestion here. Could you clarify? Thanks

**Review by NuriaQueralt — CHANGES_REQUESTED** [review]
Hi @ljgarcia I requested some minor information before merging your PR. If you could provide it, this would be helpful to review your suggestions. Thanks!

File: projects/36/bhxiv/paper.md (hunk 1 above)
Comment (ljgarcia) [reply projects/36/bhxiv/paper.md:107]: It was OMOP model --> changed to Observational Medical Outcomes Partnership (OMOP) Common Data Model

File: projects/36/bhxiv/paper.md (hunk 4 above)
Comment (ljgarcia) [reply projects/36/bhxiv/paper.md:301]: organizations --> organisations

File: projects/36/bhxiv/paper.md (hunk 5 above)
Comment (ljgarcia) [reply projects/36/bhxiv/paper.md:303]: projects/organizations --> projects/organisations

File: projects/36/bhxiv/paper.md (hunk 4 above)
Comment (ljgarcia) [reply projects/36/bhxiv/paper.md:301]: recognized --> recognised
By the way "endeavors" should be "endeavours"

File: projects/36/bhxiv/paper.md (hunk 2 above)
Comment (ljgarcia) [reply projects/36/bhxiv/paper.md:280]: optimization --> optimisation

File: projects/36/bhxiv/paper.md (hunk 1 above)
Comment (NuriaQueralt) [reply projects/36/bhxiv/paper.md:107]: Thanks, I will add also the CDM acronym.

File: projects/36/bhxiv/paper.md (hunk 2 above)
Comment (NuriaQueralt) [reply projects/36/bhxiv/paper.md:280]: Thanks!

File: projects/36/bhxiv/paper.md (hunk 4 above)
Comment (NuriaQueralt) [reply projects/36/bhxiv/paper.md:301]: Thanks! I will change endeavours as well

File: projects/36/bhxiv/paper.md (hunk 5 above)
Comment (NuriaQueralt) [reply projects/36/bhxiv/paper.md:303]: Thanks!

**Comment by NuriaQueralt** [comment]
Hi @ljgarcia, in some suggestions on Github I only saw the full paragraphs in red that's why I asked (in some others the change was explicitly highlighted). I don't know why the difference. Anyways I accepted your PR now. Thanks!

**Pull request merged**